### PR TITLE
Feature/14179 ess UI include upload explanation text

### PR DESCRIPTION
**Pull request by prasad14878**

### CODE REVIEW

**Review by JiviteshT — APPROVED** [review]
Looks good.

File: Tests/FunctionalTests/ess-ui-home.spec.ts [hunk 1]
```diff
@@ -56,11 +56,11 @@ test.describe('ESS UI Home Page Functional Test Scenarios', () => {
     });
 
     // https://dev.azure.com/ukhocustomer/File-Share-Service/_workitems/edit/13796
-    test('Verify appending "/exchangesets" in  url after login, navigates to ESS landing page', async ({ page }) => {
+    test.only('Verify appending "/exchangesets" in  url after login, navigates to ESS landing page', async ({ page }) => {
```
[the remark this answers is not in the document]
Comment (prasad14878) [reply Tests/FunctionalTests/ess-ui-home.spec.ts]: Resolved

File: src/app/features/exchange-set/ess-upload-file/ess-upload-file.component.html [hunk 2]
```diff
@@ -6,10 +6,23 @@ <h3 aria-live="polite" role="alert">{{messageDesc}}</h3>
...
+    
+            <p class="light">Once you have uploaded a list, you can then make an exchange set containing a 
+                maximum of {{maxEncSelectionLimit}} individual ENCs.
+            </p>
+        </div>        
 
         <ukho-file-input class="UploadFileIput" id="file-upload" (drop)="uploadListener($event)" (change)="uploadListener($event)" accept=".txt,.csv"></ukho-file-input>
 
-        <ukho-button class="UploadButton" (click)="loadFileReader()">Proceed</ukho-button>
+        <ukho-button  (click)="loadFileReader()">Proceed</ukho-button>
```
Comment (rvijayavel) [inline src/app/features/exchange-set/ess-upload-file/ess-upload-file.component.html:25]: Don't we need 'UploadButton' class?. It is specified in scss file. If it is not used, kindly remove it from scss file also.

**Review by rvijayavel — APPROVED** [review]
Other than 2 minor comments, code looks good.

File: src/app/features/exchange-set/ess-upload-file/ess-upload-file.component.html (hunk 2 above)
Comment (prasad14878) [reply src/app/features/exchange-set/ess-upload-file/ess-upload-file.component.html:25]: Resolved

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**